.claude/tdd-bdd-hybrid.md

```
@@ -0,0 +1,294 @@
+       // TDD cycle for domain logic
+   }
+   
+   // api/internal/application/review_service_test.go
+   func TestReviewService_Create(t *testing.T) {
+       // TDD cycle for service layer
+   }
+   ```
+
+## LAYER-SPECIFIC STRATEGIES
+
+### Interface Layer (Handlers/Controllers)
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)